automation/tdx_canonical_setup.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+CUR_DIR=$(pwd)
+TDX_DIR="$CUR_DIR"/tdx_setup
+GUEST_TOOLS_DIR="$TDX_DIR"/guest-tools
+GUEST_IMG_DIR="$GUEST_TOOLS_DIR"/image
+ATTESTATION_DIR="$TDX_DIR"/attestation
+BRANCH_NAME=noble-24.04
+
+clone_tdx_repo(){
+        echo -e "\n\nCloning TDX git repository with '$BRANCH_NAME' branch.."
+        [ -d $TDX_DIR ] && rm -rf $TDX_DIR
+        git clone -b $BRANCH_NAME https://github.com/canonical/tdx.git $TDX_DIR
+}
+
+check_production_system() {
+        cd $ATTESTATION_DIR
+        echo -e "\n\nVerifying if the system supports attestation.."
+        output=$(sudo ./check-production.sh)
+        echo $output
+        if [[ $output =~ "Production" ]]; then
+                echo -e "Attestation is supported.\nThe attestation components will be installed "

```

Comment:
How about "Intel® SGX Data Center Attestation Primitives (Intel® SGX DCAP) will be installed on the Host"

Reply:
updated the echo statement with new commit